Lombiq.Hosting.Tenants.Maintenance/Readme.md

```diff
@@ -62,7 +62,8 @@ The following configuration options are available to set the site URL:
       "UpdateSiteUrl": {
         "IsEnabled": true,
         "SiteUrl": "https://domain.com/{TenantName}",
-        "DefaultTenantSiteUrl": "https://domain.com"
+        "DefaultTenantSiteUrl": "https://domain.com",
+        "SiteUrlFromTenantName": ""
```

Comment:
If this option is not required, we can hide it when not used, so you can remove this line.